js/Heap.js

```diff
@@ -21,11 +22,15 @@ const track = bailOnError((event, payload) => {
     RNHeap.track(event, flatten(payload));
 
     checkDisplayNamePlugin();
+
+    console.log('Heap.track call:', event, payload);
   } catch (e) {
     console.log('Error calling Heap.track\n', e);
   }
 });
 
+export { HeapIgnore, HeapCaptureRestrictor };
```

Comment:
I like the way you set this up... Could I actually use this as `<Heap.Ignore ...` or does that not work?  I think I like the way that looks slightly better, but I also like that it could be accessed either way.

Reply:
Yea, the idea is that 
```
import { HeapIgnore } rom '@heap/react-native-heap';
...
<HeapIgnore>
```
```
import Heap rom '@heap/react-native-heap';
...
<Heap.Ignore>
```
are the same thing.